make/harbor.yml

```diff
@@ -10,12 +10,12 @@ http:
   port: 80
 
 # https related config
-# https:
-#   # https port for harbor, default is 443
-#   port: 443
-#   # The path of cert and key files for nginx
-#   certificate: /your/certificate/path
-#   private_key: /your/private/key/path
+https:
+  # https port for harbor, default is 443
+  port: 443
+  # The path of cert and key files for nginx
+  certificate: /data/cert/server.crt
```

Comment:
Why not keep it as is?

Reply:
+1 

or

`/PATH_TO_YOUR_CERTIFICATE`
`/PATH_TO_YOUR_PRIVATEKEY`

Reply:
But, if we change back, do you need to add an installation check in the install.sh?

And, also, we need to update the travis and nightly accordingly?